Assets/_BForBoss/_LevelDesign/Managers/WorldManager.cs

```diff
@@ -0,0 +1,59 @@
+using UnityEngine;
+
+namespace BForBoss
+{
+    public class WorldManager : MonoBehaviour
+    {
+        private StateManager _stateManagerInstance = StateManager.Instance;
```

Comment:
NIT: probably don't need to name it `_stateManagerInstance `
just name it `_stateManager`